src/include/execution/compiler/operator/output_translator.h

```diff
@@ -37,18 +37,23 @@ class OutputTranslator : public OperatorTranslator {
    */
   void DefineHelperStructs(util::RegionVector<ast::StructDecl *> *decls) override;
 
-  /** Set up counters for Lin's models. */
-  void InitializeQueryState(FunctionBuilder *function) const override;
-
   /**
-   * Perform the main work of the translator.
+   * @param pipeline The current pipeline.
```

Comment:
I suppose these comments aren't actually necessary since they don't add anything over the override (if we're going to be consistent with TeardownPipelineState not having any)